### DISCUSSION TITLE
bagaimana caranya membuat (syntax) tambahan kolom baru dalam df

### DISCUSSION THREAD

**Discussion by Imam-Danu** [teamalgoritma/community#13] [Q&A] [answered]
misalkan membuat Total belanja dengan menggunkan kolom harga barang + jumlah barang

**Comment by onlyphantom** [comment, marked as answer]
Ini cara membuat kolom `total_belanja` di dataframe `df` nya memakai kolum bernama `harga` dan `jumlah`:

```
df['total_belanja'] = df['harga'] * df['jumlah'] 
df['total_belanja'] = df['harga'] + df['jumlah'] 
```

**Comment by muhrisman** [comment]
Halo Pak Danu!

Untuk membuat kolom tambahan dalam suatu dataframe (df) menggunakan python bapak bisa menggunakan kode berikut:

df['Total_belanja'] = df['harga_barang'] * df['jumlah_barang']

Semoga membantu :)
